frontend/src/components/games/list/GameListItem.tsx

```diff
@@ -153,6 +235,38 @@ export function RenderResult(params: GridRenderCellParams) {
     );
 }
 
+function parseTimeControl(
```

Comment:
Parsing a time control is actually more complicated than this, but we can probably use this for the majority of games that are being submitted by users. There's probably some master games where this won't be accurate, but not a big deal

Reply:
Do we have examples of the logic elsewhere? I know there are multiple time increments... The logic here is specifically for Lichess and I'm wondering if I should check that and if it's not Lichess use verbatim?

Reply:
You can look at the Clock Usage underboard tab.

Also, a full description of the TimeControl header is here: http://www.saremba.de/chessgml/standards/pgn/pgn-complete.htm#c9.6.1

Again, the vast majority of games will have a simple time control header, so this is pretty low priority

Reply:
Okay, I didn't mess too much with it, but looking at ClockUsage showed me there was existing functions I could use so I dropped them in.